frontend/src/components/Pages/FrontPage/AutoScheduleSection/AutoScheduleSection.tsx

```diff
@@ -166,53 +174,94 @@ const AutoScheduleList = () => {
             )
         })
 
-    return (
+    const AddScheduleNewMissionButton = () => (
+        <AlignedTextButton onClick={openDialog}>
+            <Icon name={Icons.Add} size={24} />
+            {TranslateText('New scheduled mission')}
+        </AlignedTextButton>
+    )
+
+    const DisplayScheduledMissions = () => (
         <>
-            <StyledButtonSection>
-                <StyledButton onClick={openDialog}>
-                    <StyledIcon name={Icons.Add} size={24} />
-                    {TranslateText('New scheduled mission')}
-                </StyledButton>
-            </StyledButtonSection>
-            {autoScheduleMissionDefinitions.length > 0 && (
-                <StyledSection>
-                    <StyledHeader>
-                        <Typography>
-                            {TranslateText('These missions will be automatically scheduled at the specified time')}
-                        </Typography>
-                    </StyledHeader>
+            {autoScheduleMissionDefinitions.length > 0 ? (
+                <>
+                    <Typography>
+                        {TranslateText('These missions will be automatically scheduled at the specified time')}
+                    </Typography>
                     <StyledDayOverview>
                         <DayOverview />
                     </StyledDayOverview>
-                    {dialogOpen && (
-                        <StyledDialog open={true}>
+                </>
+            ) : (
+                <Typography>{TranslateText('There are currently no automatically scheduled missions.')}</Typography>
+            )}
+        </>
+    )
+
+    const SelectMission = () => (
```

Comment:
```suggestion
    const SelectMissionDialog = () => (
```
This is slightly more descriptive

Reply:
Renamed to SelectMissionDialogContent